test/base/TTokenRegistryIsTTokenTest.js

```diff
@@ -0,0 +1,94 @@
+// JS Libraries
+const withData = require('leche').withData;
+const { t, createMocks } = require('../utils/consts');
+const { createTestSettingsInstance } = require("../utils/settings-helper");
+const LogicVersionsRegistryEncoder = require('../utils/encoders/LogicVersionsRegistryEncoder');
+
+// Mock Contracts
+const Mock = artifacts.require('./mock/util/Mock.sol');
+
+// Smart Contracts
+const TTokenRegistry = artifacts.require('./base/TTokenRegistry.sol');
+const Settings = artifacts.require("./base/Settings.sol");
+
+contract('TTokenRegistryIsTTokenTest', function (accounts) {
+    let instance;
+    const logicVersionsRegistryEncoder = new LogicVersionsRegistryEncoder(web3);
+
+    beforeEach('Setup for each test', async () => {
+        const pauser = accounts[0];
+        
+        mocks = await createMocks(Mock, 10);
+        const versionsRegistryInstance = await Mock.new();
+        const constsInstance = await Mock.new();
+        await versionsRegistryInstance.givenMethodReturnAddress(
+            logicVersionsRegistryEncoder.encodeConsts(),
+            constsInstance.address
+        );
+
+        const pairAggregatorRegistryInstance = await Mock.new();
+        const settingsInstance = await createTestSettingsInstance(
```

Comment:
Suggestion: It would be great if we could have a function to obtain an initialized Settings instance without having to use all those params.

Reply:
Agreed